.github/workflows/actions.yml

```diff
@@ -65,6 +65,8 @@ jobs:
     - name: Install dependencies
       run: |
           pip install -r requirements.txt --progress-bar off
+          pip install torch>=2.0.1+cpu --progress-bar off
```

Comment:
I think this would also need to go into the publish to pypi action (though long term, I still favor a requirments file as a single source of truth to avoid duplication)

Reply:
yeah this feels a little janky to me as well. It's possible that we could leave these in requirements.txt for now and just have the GPU CI workflow uninstall the CPU versions before installing the GPU versions.

I'm fine either way, this just seemed like less of a headache for the GPU tests.